docs/release-notes/feature-clean-up.rst

```diff
@@ -10,10 +10,10 @@ Agent Resource Manager:
 
 -  Container Runtimes: We will limit support to Docker for Agent Resource Managers.
 
--  Job Scheduling: Moving a job will require adjusting its priority; jobs cannot be shifted within
-   the same priority group. Support for round-robin and fair share schedulers is discontinued. We
-   recommend using the priority scheduler, as it meets most scheduling needs and simplifies
-   configuration.
+-  Job Scheduling: Switching to ``priority`` for the default scheduler. Support for round-robin and
+   fair share schedulers is discontinued. We recommend using the priority scheduler, as it meets
+   most scheduling needs and simplifies configuration. Moving a job will require adjusting its
+   priority; jobs cannot be shifted within the same priority group.
```

Comment:
-  Job Scheduling: The default scheduler is now ``priority``. Support for the round-robin and fair share schedulers has been discontinued. We recommend using the priority scheduler, as it meets
   most scheduling needs and simplifies configuration. To move a job, you will need to adjust its priority; jobs cannot be shifted within the same priority group.